route-rs-runtime/src/link/mod.rs

```diff
@@ -31,25 +31,35 @@ pub use self::tee_link::*;
 /// Drops all packets that are ingressed, asynchronous.
 mod blackhole_link;
 pub use self::blackhole_link::*;
+use crate::element::Element;
 
 /// All Links communicate through streams of packets. This allows them to be composable.
 pub type PacketStream<Input> = Box<dyn Stream<Item = Input, Error = ()> + 'static + Send>;
 /// Some Links may need to be driven by Tokio. This represents a handle to something Tokio can run.
 pub type TokioRunnable = Box<dyn Future<Item = (), Error = ()> + 'static + Send>;
+/// LinkBuilders build this.
+pub type Link<Output> = (Vec<TokioRunnable>, Vec<PacketStream<Output>>);
```

Comment:
Better name pls?

Reply:
I mean, it isn't the worst name right now.  LinkBuilder's build Links, that is pretty clear? Still open to ideas though.